src/app/graphql/resolvers.ts

```diff
@@ -1,8 +1,18 @@
-import { prisma } from '../client/client';
-import { UserInput } from '../interfaces';
+import prisma from '../client/client';
+import { UserInput } from '../interfaces/user';
+import { LoginInput } from '../interfaces/login';
 import isPasswordValid from './password';
 import bcrypt from 'bcrypt';
 import { CustomError } from '../errors/CustomError';
+import jwt from 'jsonwebtoken';
+import 'dotenv/config';
+import { AuthenticationError } from 'apollo-server';
+const JWT_SECRET = process.env.JWT_SECRET ?? '';
+
+const comparePassword = async (password: string, hashedPassword: string) => {
+  const isMatch = await bcrypt.compare(password, hashedPassword);
+  return isMatch;
+};
```

Comment:
poderia tipar o retorno da função
```suggestion
const comparePassword = async (password: string, hashedPassword: string): Promise<boolean> => {
  return bcrypt.compare(password, hashedPassword);
};
```